### ISSUE TITLE
Error: Cannot read properties of undefined (reading 'text')

### ISSUE THREAD

**Issue by nrayann** [nestjs/nest#13820]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current behavior

Hello, I've just created a new project and I'm trying to follow the Authentication docs.

When I run `nest g module auth` I get the following error:

```
Error: Cannot read properties of undefined (reading 'text')

Failed to execute command: node @nestjs/schematics:module --name=auth --no-dry-run --no-skip-import --language="ts" --source-root="src" --spec --no-flat --spec-file-suffix="spec"
```

### Minimum reproduction code

https://github.com/nestjs/nest

### Steps to reproduce

$ npm i -g @nestjs/cli
$ nest new project-name
$ nest g module auth

### Expected behavior

I expected the file to be created.

### Package

- [ ] I don't know. Or some 3rd-party package
- [ ] <code>@nestjs/common</code>
- [ ] <code>@nestjs/core</code>
- [ ] <code>@nestjs/microservices</code>
- [ ] <code>@nestjs/platform-express</code>
- [ ] <code>@nestjs/platform-fastify</code>
- [ ] <code>@nestjs/platform-socket.io</code>
- [ ] <code>@nestjs/platform-ws</code>
- [ ] <code>@nestjs/testing</code>
- [ ] <code>@nestjs/websockets</code>
- [ ] Other (see below)

### NestJS version

10.3.10

### Packages versions

[System Information]
OS Version     : Linux 5.15.0-116-generic
NodeJS Version : v20.15.1
YARN Version    : 1.22.19 

[Nest CLI]
Nest CLI Version : 10.4.2 

[Nest Platform Information]
platform-express version : 10.3.10
schematics version       : 10.1.2
testing version          : 10.3.10
common version           : 10.3.10
core version             : 10.3.10
cli version              : 10.4.2

### Node.js version

20.15.1

### In which operating systems have you tested?

- [ ] macOS
- [ ] Windows
- [X] Linux

**Comment by maxim-cpg** [comment]
Same here on:
node v21.7.2
nest 10.4.2

**Comment by maxim-cpg** [comment]
After some digging found that it's TS version. When we updated to `5.5.4` it causes this error with nest generators. 
With `5.5.3` it works fine.

**Comment by bishwoconpago** [comment]
works for me too.. thanks @maxim-cpg

**Comment by nrayann** [comment]
works for me too.. thanks @maxim-cpg

**Comment by dreamsachiever90** [comment]
Worked like charm. Thanks

**Comment by micalevisk** [comment]
duplicate https://github.com/nestjs/nest-cli/issues/2662

**Comment by alexlionnel** [comment]
I have the same issue:
Node: v20.13.1
Nest: 10.4.2
typescripot: 5.5.3

I tried to execute this command
```nest generate module user```

but i got the same error
```
Error: Cannot read properties of undefined (reading 'text')

Failed to execute command: node @nestjs/schematics:module --name=user --no-dry-run --no-skip-import --language="ts" --source-root="src" --spec --no-flat --spec-file-suffix="spec"
```

**Comment by kamilmysliwiec** [comment]
Fixed in `@nestjs/schematics@10.1.3`
